src/screens/Asset/Asset.js

```diff
@@ -343,6 +351,7 @@ class AssetScreen extends React.Component<Props, State> {
     });
     const relatedTransactions = isSynthetic ? ppnTransactions : mainnetTransactions;
     const isSupportedByExchange = exchangeSupportedAssets.some(({ symbol }) => symbol === token);
+    const isKeyBasedActive = !!activeAccount && activeAccount.type === ACCOUNT_TYPES.KEY_BASED;
```

Comment:
Minor: `const isKeyBasedActive = !isSmartAccount;`
and in createStructuredSelector:
`isSmartAccount: isActiveAccountSmartWalletSelector,`